docs/index.rst

```diff
@@ -118,7 +118,7 @@ contacting support.
 +--------------------------------------------------+---------------+----------------+
 | :ref:`pyramid`                                   | >= 1.7        | No             |
 +--------------------------------------------------+---------------+----------------+
-| :ref:`pytest`                                    | >= 3.0        | Yes            |
+| :ref:`pytest`                                    | >= 3.6.0      | Yes            |
```

Comment:
```suggestion
| :ref:`pytest`                                    | >= 4.0.0      | Yes            |
```

It looks like we actually only test with 4 and above

Reply:
@Kyle-Verhoog Thanks! I was going off this:
https://github.com/DataDog/dd-trace-py/blob/eef75c3117dc378e9d548a95ea2291819e934757/tox.ini#L317-L318

is that incorrect?

Reply:
Ah yeah those are just the environment definitions (tox files are 🤯 to read). These are the lines that specify the environments for the pytest integration tests: https://github.com/DataDog/dd-trace-py/blob/eef75c3117dc378e9d548a95ea2291819e934757/tox.ini#L102-L103